plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/model/DependencyEdgeType.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2017 Marcus Mews.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Marcus Mews - Initial API and implementation
+ */
+package org.eclipse.n4js.flowgraphs.model;
+
+//TODO GH-235
+@SuppressWarnings("javadoc")
+public enum DependencyEdgeType {
+
+	op, ad, fd, se, dod, od, cd, io, init
```

Comment:
I know this enum is still a TODO, but the enum values are hard to understand. Or are they standard values in the literature?